core/src/navigation/services/semi-collapsed-navigation.js

```diff
@@ -10,16 +10,15 @@ class SemiCollapsibleNavigationClass {
     let currentCollapsedState = localStorage.getItem(
       NavigationHelpers.COL_NAV_KEY
     );
-    let checkTheResponsiveNavSettings =
+    let getResponsiveNavSettings =
```

Comment:
It seems using `const` is more suitable. There is no updated afterward.
I would suggest name it with `is-` prefix. e.g. `const isResponsiveNavSemiCollapsibleOrFiori3`  something like this. It is easier to understand at first sight.

Reply:
done